core_impl/src/main/java/io/opencensus/implcore/stats/IntervalBucket.java

```diff
@@ -75,7 +75,7 @@ void record(List<TagValue> tagValues, double value) {
    */
   double getFraction(Timestamp now) {
     Duration elapsedTime = now.subtractTimestamp(start);
-    checkArgument(toMillis(elapsedTime) >= 0 && toMillis(elapsedTime) < toMillis(duration),
+    checkArgument(toMillis(elapsedTime) >= 0 && elapsedTime.compareLength(duration) < 0,
```

Comment:
`toMillis(elapsedTime) >= 0` could compare to a zero-length Duration.

Reply:
Done.